src/components/MessagesList/MessagesList.vue

```diff
@@ -329,40 +352,89 @@ export default {
 
 	methods: {
 		prepareMessagesGroups(messages) {
-			const groups = []
+			let prevGroupMap = {}
```

Comment:
but still need to check if `prevGroupMap` is truthy anyway . Although it is false which is the first time, it will cause an error when accessing keys of undefined.

Reply:
yes, just `prevGroupMap` without Object constructor